Objective-C/CBLCollection.h

```diff
@@ -87,7 +87,8 @@ extern NSString* const kCBLDefaultCollectionName;
  @param error On return, the error if any.
  @return The CBLDocument object.
  */
-- (nullable CBLDocument*) documentWithID: (NSString*)documentID error: (NSError**)error;
+- (nullable CBLDocument*) documentWithID: (NSString*)documentID
+                                   error: (NSError**)error NS_SWIFT_NOTHROW;
```

Comment:
I do not know that there is NS_SWIFT_NOTHROW.

Reply:
We are already using the same in [isDocumentPending API](https://github.com/couchbase/couchbase-lite-ios/blob/master/Objective-C/CBLReplicator.h#L181)